tests/js/host-test.coffee

```diff
@@ -34,3 +36,21 @@ describe 'Annotator.Host', ->
         assert.isFalse(host.visibleHighlights)
         done()
       host.publish('panelReady')
+
+  describe 'crossframe listeners', ->
+    emitHostEvent = (event, args...) ->
+      fn(args...) for [evt, fn] in fakeCrossFrame.on.args when event == evt
```

Comment:
I'm not sure about testing events like this. This is assuming things about the internals of `.on(...)`, which seems fragile. I'd prefer it if you just checked that `.on(...)` was called with the right arguments (i.e.call the callback function and check it calls the target you expect)/